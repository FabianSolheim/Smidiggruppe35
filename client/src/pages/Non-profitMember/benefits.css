.containerBenefits{
    width: 100%;
    display: flex;
    align-items: center;
}

.widthContainerBenefits {
    width: 1440px;
    margin: auto;
}
.joinUsBtnBenefits {
    font-family: "Montserrat", sans-serif;
    border-radius: 0;
    height: 50px;
    width: 200px;
    background-color: #034F7A;
    color: white;
    border: 0;
    font-size: 16px;
    display:block; margin: 0 auto
}
.benefitsImg {
    display: block;
    margin-left: auto;
    margin-right: auto;
}
.textBenefits {
    text-align: left;
}
@media only screen and (max-width: 900px) {

    .benefitsImg {
        width: 350px;
        margin-top: 20px;
    }
    .textBenefits {
        text-align: center;
    }

}

