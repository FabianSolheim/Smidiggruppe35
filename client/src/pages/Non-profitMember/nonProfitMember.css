body{
    margin: 0;
}
.container {
    max-height: 1130px;
    max-width: 1440px;
    margin: auto;
    width: 90%;
}
.box {
    max-height: 1130px;
    width: 1440px;
    margin: auto;
    background-color: white;
}
.header {
    width: 100%;
    height: 500px;
    background-image: url("https://images.unsplash.com/photo-1505455184862-554165e5f6ba?ixlib=rb-1.2.1&raw_url=true&q=80&fm=jpg&crop=entropy&cs=tinysrgb&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=2969");
    background-repeat: no-repeat;
    background-size: cover;
    background-position: center;
    text-align: center;
    font-size: 64px;
}
.header em {
    color: #034F7A;
    font-style: normal;
}
.top {
    font-size: 0px;
}
.headline {
    margin-top: 200px;
    font-size: 64px;
    font-family: "Montserrat", sans-serif;
    color: white;
}
.description {
    font-size: 30px;
    margin-right: 115px;
    font-family: "Montserrat", sans-serif;
    color: white;
}
.description em {
    color: #034F7A;
    font-style: normal;
}
.joinUsBtn {
    font-family: "Montserrat", sans-serif;
    border-radius: 0;
    height: 50px;
    width: 200px;
    background-color: #034F7A;
    color: white;
    border: 0;
    font-size: 16px;
}


@media only screen and (max-width: 750px) {


    .headline {
        font-size: 34px;
        margin-top: 100px;
    }
    .description {
        font-size: 16px;
    }
    .consultationBtn {
        font-size: 16px;
        margin-top: 60px;
    }
    .header {
        max-height: 5em;
    }

}