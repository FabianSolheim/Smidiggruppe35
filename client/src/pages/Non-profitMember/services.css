.containerServices {
    background-color: rgba(3,79,122,0.1);
    width: 100%;
    height: 295px;
    display: flex;
    align-items: center;
}
.widthContainer {
    width: 1440px;
    margin: auto;
}