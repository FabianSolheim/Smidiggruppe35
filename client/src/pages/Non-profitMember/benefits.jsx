import React from "react";
import { Grid, Box, Container } from "@mui/material";
import "./benefits.css";
import { HeadlineWithUnderline } from "../../components/HeadlineWithUnderline/headlineWithUnderline";
import { BtnWithHeadline } from "../../components/BtnWithHeadline/BtnWithHeadline";

export function Benefits() {
  return (
    <>
      {HeadlineWithUnderline("Benefits", 100, "#034F7A", 50, "#212121", "36px")}
      <div className="containerBenefits">
        <div className="widthContainerBenefits">
          <Box sx={{ flexGrow: 1 }}>
            <Container>
              <Grid
                container
                spacing={0}
                justifyContent="center"
                style={{ maxWidth: 1440, margin: "auto" }}
              >
                <Grid item xs={8} sm={6}>
                  <p>
                    <strong>
                      More nonprofits are going digital: payments, platforms,
                      marketing and storytelling are done and supported online.
                    </strong>
                    <br />
                    <br />
                    Individual donations via digital platforms have increased by
                    45% in 2020
                    <br /> in Norway alone.
                    <br />
                    <br />
                    This trend means that nonprofits need to spend time and
                    money to
                    <br /> develop those new solutions. Our platform gives the
                    nonprofits exactly
                    <br /> these tools (and more) for free, and ensures that
                    also smaller nonprofits
                    <br /> can be a part of the digitization without using up
                    their own resources.
                  </p>
                </Grid>
                <Grid item xs={12} sm={6}>
                  <div>
                    <img
                      className="benefitsImg"
                      src={require("../../images/nonmemberprofit.svg")}
                    />
                  </div>
                </Grid>
                <Grid item xs={12} md={10}>
                  <div className="joinUsDiv">
                    {BtnWithHeadline("Ready to get started?", "Join us")}
                  </div>
                </Grid>
              </Grid>
            </Container>
          </Box>
        </div>
      </div>
    </>
  );
}
