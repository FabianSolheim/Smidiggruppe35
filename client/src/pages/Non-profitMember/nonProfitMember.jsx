import React from "react";
import "./nonProfitMember.css";
import { Benefits } from "./benefits";
import { Services } from "./services";
import { HeaderBox } from "../../components/HeaderBox/HeaderBox";
export function NonProfit() {
  return (
    <>
        {HeaderBox(
            "https://images.unsplash.com/photo-1505455184862-554165e5f6ba?crop=entropy&cs=tinysrgb&fm=jpg&ixlib=rb-1.2.1&q=80&raw_url=true&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=2969",
            <h1>Non-profit <em>members</em></h1>,
            "white",
            <p>There's often no solid <strong><em>philanthropic</em></strong> strategy and <strong><em>nonprofits</em></strong> are picked on an <strong><em>ad-hoc basis.</em></strong></p>,
            "white"
        )}
      <div style={{ height: 100 }} />
      <Benefits />
      <div style={{ height: 100 }} />
      <Services />
      <div style={{ height: 100 }} />
      <img src={require("../../images/Footer.svg")} width={"100%"} />
    </>
  );
}
