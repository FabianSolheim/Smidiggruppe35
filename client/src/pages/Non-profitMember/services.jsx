import React from "react";
import Box from "@mui/material/Box";
import Grid from "@mui/material/Grid";
import "./services.css";
import { HeadlineWithUnderline } from "../../components/HeadlineWithUnderline/headlineWithUnderline";

export function Services() {
  return (
    <>
      {HeadlineWithUnderline("Services", 100, 50, "#212121", "36px")}
      <div className="containerServices">
        <div className="widthContainer">
          <Box sx={{ flexGrow: 1 }}>
            <Grid container spacing={2}>
              <Grid item xs={12} md={4}>
                <div style={{ textAlign: "center", marginTop: 10 }}>
                  <img src={require("../../images/foundingicon.svg")} />
                  <p
                    style={{
                      fontSize: 18,
                      color: "#490374",
                      marginTop: 10,
                    }}
                  >
                    <strong>Funding</strong>
                  </p>
                  <p
                    style={{
                      fontSize: 16,
                      marginTop: 8,
                    }}
                  >
                    <strong>Regular</strong> funding
                    <br /> stream
                  </p>
                </div>
              </Grid>
              <Grid item xs={12} md={4}>
                <div style={{ textAlign: "center", margin: 20 }}>
                  <img src={require("../../images/platfromicon.svg")} />
                  <p
                    style={{
                      fontSize: 18,
                      color: "#490374",
                      marginTop: 10,
                    }}
                  >
                    <strong>Platform</strong>
                  </p>
                  <p
                    style={{
                      fontSize: 16,
                      marginTop: 8,
                    }}
                  >
                    One platform <strong>connecting</strong>
                    <br />
                    them to their
                    <br />
                    <strong>donors</strong>
                  </p>
                </div>
              </Grid>
              <Grid item xs={12} md={4}>
                <div style={{ textAlign: "center", margin: 20 }}>
                  <img src={require("../../images/toolsicon.svg")} />
                  <p
                    style={{
                      fontSize: 18,
                      color: "#490374",
                      marginTop: 10,
                    }}
                  >
                    <strong>Tools</strong>
                  </p>
                  <p
                    style={{
                      fontSize: 16,
                      marginTop: 8,
                    }}
                  >
                    <strong>Free</strong> digital
                    <br /> tools
                  </p>
                </div>
              </Grid>
            </Grid>
          </Box>
        </div>
      </div>
    </>
  );
}
