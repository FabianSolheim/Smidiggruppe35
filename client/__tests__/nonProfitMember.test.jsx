
import React from "react";
import ReactDOM from "react-dom";
import {NonProfit} from "../src/pages/Non-profitMember/nonProfitMember";


describe("Nonprofit page", () => {
    it("shows nonprofit page", () => {
        const domElement = document.createElement("div");
        ReactDOM.render(<NonProfit />, domElement);
        expect(domElement.innerHTML).toMatchSnapshot();
    });
});