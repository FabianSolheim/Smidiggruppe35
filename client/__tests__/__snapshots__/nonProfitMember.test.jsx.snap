// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nonprofit page shows nonprofit page 1`] = `"<div class=\\"TopHeader\\" style=\\"background-image: url(https://images.unsplash.com/photo-1505455184862-554165e5f6ba?crop=entropy&amp;cs=tinysrgb&amp;fm=jpg&amp;ixlib=rb-1.2.1&amp;q=80&amp;raw_url=true&amp;ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&amp;auto=format&amp;fit=crop&amp;w=2969); background-repeat: no-repeat; background-size: cover; background-position: center; text-align: center;\\"><div class=\\"HeaderBox\\"><div class=\\"BoxHeadline\\" style=\\"color: white;\\"><div>Non-profit <em>members</em></div></div><div class=\\"BoxText\\" style=\\"color: white;\\"><div>There's often no solid <strong><em>philanthropic</em></strong> strategy and <strong><em>nonprofits</em></strong> are picked on an <strong><em>ad-hoc basis.</em></strong></div></div></div></div><div style=\\"height: 100px;\\"></div><div><div style=\\"font-size: 36px; font-weight: bold; text-align: center; color: rgb(33, 33, 33);\\">Benefits</div></div><div style=\\"width: 100px; height: 2px; background-color: rgb(3, 79, 122); margin: 5px auto 50px auto;\\"></div><div class=\\"containerBenefits\\"><div class=\\"widthContainerBenefits\\"><div class=\\"MuiBox-root css-i9gxme\\"><div class=\\"MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root\\"><div class=\\"MuiGrid-root MuiGrid-container css-1lym95h-MuiGrid-root\\" style=\\"max-width: 1440px; margin: auto;\\"><div class=\\"MuiGrid-root MuiGrid-item MuiGrid-grid-xs-8 MuiGrid-grid-sm-6 css-72ujfr-MuiGrid-root\\"><p><strong>More nonprofits are going digital: payments, platforms, marketing and storytelling are done and supported online.</strong><br><br>Individual donations via digital platforms have increased by 45% in 2020<br> in Norway alone.<br><br>This trend means that nonprofits need to spend time and money to<br> develop those new solutions. Our platform gives the nonprofits exactly<br> these tools (and more) for free, and ensures that also smaller nonprofits<br> can be a part of the digitization without using up their own resources.</p></div><div class=\\"MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 css-1b3l6lk-MuiGrid-root\\"><div><img class=\\"benefitsImg\\" src=\\"test-file-stub\\"></div></div><div class=\\"MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-10 css-1yysv6i-MuiGrid-root\\"><div class=\\"joinUsDiv\\"><div><h1 class=\\"headlineText\\">Ready to get started?</h1><button class=\\"buttonUnderHeadline\\">Join us</button></div></div></div></div></div></div></div></div><div style=\\"height: 100px;\\"></div><div><div style=\\"font-size: 36px; font-weight: bold; text-align: center; color: rgb(33, 33, 33);\\">Services</div></div><div style=\\"width: 100px; height: 2px; background-color: rgb(3, 79, 122); margin: 5px auto 50px auto;\\"></div><div class=\\"containerServices\\"><div class=\\"widthContainer\\"><div class=\\"MuiBox-root css-i9gxme\\"><div class=\\"MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 css-mhc70k-MuiGrid-root\\"><div class=\\"MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4 css-1lj5egr-MuiGrid-root\\"><div style=\\"text-align: center; margin-top: 10px;\\"><img src=\\"test-file-stub\\"><p style=\\"font-size: 18px; color: rgb(3, 79, 122); margin-top: 10px;\\"><strong>Funding</strong></p><p style=\\"font-size: 16px; margin-top: 8px;\\"><strong>Regular</strong> funding<br> stream</p></div></div><div class=\\"MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4 css-1lj5egr-MuiGrid-root\\"><div style=\\"text-align: center; margin: 20px;\\"><img src=\\"test-file-stub\\"><p style=\\"font-size: 18px; color: rgb(3, 79, 122); margin-top: 10px;\\"><strong>Platform</strong></p><p style=\\"font-size: 16px; margin-top: 8px;\\">One platform <strong>connecting</strong><br>them to their<br><strong>donors</strong></p></div></div><div class=\\"MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4 css-1lj5egr-MuiGrid-root\\"><div style=\\"text-align: center; margin: 20px;\\"><img src=\\"test-file-stub\\"><p style=\\"font-size: 18px; color: rgb(3, 79, 122); margin-top: 10px;\\"><strong>Tools</strong></p><p style=\\"font-size: 16px; margin-top: 8px;\\"><strong>Free</strong> digital<br> tools</p></div></div></div></div></div></div><div style=\\"height: 100px;\\"></div>"`;
